operate/client/src/modules/queries/useProcessInstancesStatistics.test.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {renderHook, waitFor} from '@testing-library/react';
+import {QueryClient, QueryClientProvider} from '@tanstack/react-query';
+import {useProcessInstancesStatistics} from './useProcessInstancesStatistics';
+import {mockFetchProcessInstancesStatistics} from 'modules/mocks/api/v2/processInstances/fetchProcessInstancesStatistics';
+
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      retry: 0,
+    },
+  },
+});
```

Comment:
For testing it's better to create a mock client for all I believe
At the time the recommendation was to use the client like this while testing: https://github.com/camunda/camunda/blob/main/tasklist/client/src/modules/react-query/getMockQueryClient.ts#L11-L24